tests/ProductionConfigTests.js

```diff
@@ -24,28 +24,710 @@ https://github.com/GPII/universal/blob/master/LICENSE.txt
+                        "http://registry.gpii.net/applications/org.alsa-project": {
+                            "masterVolume": 50
+                        },
+                        "http://registry.gpii.net/common/volume": 0.5,
+                        "http://registry.gpii.net/applications/org.gnome.desktop.interface": {
+                            "cursor-size": 90,
+                            "text-scaling-factor": 0.75
+                        },
+                        "http://registry.gpii.net/common/fontSize": 9,
+                        "http://registry.gpii.net/common/cursorSize": 0.9,
+                        "http://registry.gpii.net/common/backgroundColor": "black",
+                        "http://registry.gpii.net/common/fontFaceFontName": [
+                            "Comic Sans"
+                        ],
+                        "http://registry.gpii.net/common/fontFaceGenericFontFace": "sans serif",
+                        "http://registry.gpii.net/common/magnification": 1.5,
+                        "http://registry.gpii.net/common/adaptationPreference": [
+                            {
+                                "adaptationType": "caption",
+                                "language": "en"
+                            }
+                        ],
+                        "http://registry.gpii.net/common/tableOfContents": false
+                    }
+                }
+            }
+        }
+    },
+    "timestampCreated": new Date().toISOString(),
+    "timestampUpdated": null
+};
+
+gpii.tests.productionConfigTesting.carlaKey = "carla";
+gpii.tests.productionConfigTesting.carlaTokenRequestPayload = {
+    "username": gpii.tests.productionConfigTesting.carlaKey,
+    "gpiiKey": gpii.tests.productionConfigTesting.carlaKey,
+    "password": "dummy",
+    "client_id": "pilot-computer",
+    "client_secret": "pilot-computer-secret",
+    "grant_type": "password"
+};
+
+gpii.tests.productionConfigTesting.device = {
+    OS: {
+        id: "linux"
+    },
+    solutions: [{
+        "id": "org.gnome.desktop.a11y.magnifier"
+    }]
+};
+gpii.tests.productionConfigTesting.prefsUpdate = {
+    "contexts": {
+        "gpii-default": {
+            "name": "Default preferences",
+            "preferences": {
+                "http://registry.gpii.net/common/setting1": 12
+            }
+        }
+    }
+};
+
+fluid.defaults("gpii.tests.productionConfigTesting.testCaseHolder", {
+    gradeNames: ["kettle.test.testCaseHolder"],
+    components: {
+        accessTokenRequest: {
+            type: "kettle.test.request.http",
+            options: {
+                headers: {
+                    "Content-Type": "application/x-www-form-urlencoded"
+                },
+                path: "/access_token",
+                port: 9082,
+                hostname: "flowmanager",
+                method: "POST"
+            }
+        }
+    }
+});
+
+fluid.defaults("gpii.tests.cloud.oauth2.settingsGet.requests", {
```

Comment:
The only difference between this component and [its counterpart in SettingsGetTests.js](https://github.com/GPII/universal/pull/718/files#diff-64f0f413f5dc436329fe76f4c467e3c8R33) is "port" and "hostname". Hopefully when these info are moved to their corresponding config files, this component can be shared too.

The same comment applies to "gpii.tests.cloud.oauth2.settingsPut.requests" below.

Reply:
This is a great idea!  I'll keep digging on the [host name specification in the config file](https://github.com/GPII/universal/pull/718#discussion_r253535974) to get that to work.  If so, the next step will be to move the two request components to their respective shared files.